### DISCUSSION TITLE
What PVWatts version is used in the REopt web application?

### DISCUSSION THREAD

**Discussion by jacobjahnke** [NREL/REopt-API-Analysis#67] [7 - Technology Specific Questions] [answered]
What PVWatts version is used in the REopt web application? I see references in the technical manual to Version 5 but there is a PVWatts version 6 out. Which is being used?

**Comment by adfarth** [comment, marked as answer]
Hello, REopt uses PVWatts v6 (see [here](https://github.com/NREL/REopt_API/blob/16c8aab66a3ee0ca97d16e0e6581d0ae14cef8cb/reo/src/pvwatts.py#L69) ). Thank you for noting the discrepancy in the User Manual!

**Reply by jacobjahnke** [reply]
Thank you!
